tests/utils/test-data-generator.ts

```diff
@@ -166,6 +168,22 @@ export type GenerateRecordsQueryOutput = {
   message: RecordsQueryMessage;
 };
 
+export type GenerateRecordsSubscribeInput = {
+    /**
+   * Treated as `false` if not given.
+   */
```

Comment:
Nit: indentation, dumb that linter doesn't pick this up.

Reply:
Good catch with the comment indentations! Done